### ISSUE TITLE
Weapon Selection Rework

### ISSUE THREAD

**Issue by mzegar** [mzegar/sbox-TTT#31]
Scrolling shouldn't automatically pull out your weapon. You should scroll, then left-click to pull out the weapon.

**Comment by youarereadingthis** [comment]
While personally I strongly agree with this, a surprising amount of people religiously agree with the opposite.
I suggest two options:

1. 'Show weapon list while scrolling.' (Default: on)
2. 'Instantly select weapons from list while scrolling.' (Default: off)

If no. 1 is selected then instant selection is always in effect.

**Comment by Fires1** [comment]
I agree. Keep it to two easily togglable options. Keyboard keys override and will instant select so you have options for fast weapon changes (also togglable?)

**Comment by mzegar** [comment]
Yeah, a user setting would be good for this. The reason you don't want to pull out every weapon on the scroll is you could reveal a Traitor weapon.
